common/src/indexer_service/http/config.rs

```diff
@@ -71,4 +72,5 @@ pub struct IndexerConfig {
 pub struct ScalarConfig {
     pub chain_id: u64,
     pub receipts_verifier_address: Address,
+    pub timestamp_error_tolerance: Duration,
```

Comment:
@aasseman do you know if it's possible to set default values for this?

Reply:
The most important thing is that it has to be smaller than `rav_request_timestamp_buffer_ms`

Reply:
I think we should increase the default value of `rav_request_timestamp_buffer_ms` to maybe 30sec or 1min?
Then for ex if we choose 1 min, we could have `timestamp_error_tolerance` default to 30 sec.